brokenspoke_analyzer/cli/root.py

```diff
@@ -55,8 +59,43 @@ def callback(verbose: int = typer.Option(0, "--verbose", "-v", count=True)) -> N
     logger.add(sys.stdout, format=log_format, level=log_level, colorize=True)
 
 
+def _version_callback(value: bool) -> None:
+    # Get the package's version
+
+    package_version = metadata.version("brokenspoke-analyzer")
+    if value:
+        print(f"brokenspoke-analyzer version: {package_version}")
```

Comment:
Since we're using typer, we should be using `typer.echo` to display message:

```suggestion
        typer.echo(f"brokenspoke-analyzer version: {package_version}")
```

Reply:
Ah yes! :+1: